src/Conf.ml

```diff
@@ -24,6 +24,7 @@ type t =
   ; break_fun_decl: [`Wrap | `Fit_or_vertical | `Smart]
   ; break_fun_sig: [`Wrap | `Fit_or_vertical | `Smart]
   ; break_separators: [`Before | `After | `After_and_docked]
+  ; break_sequence_separators: [`Before | `After]
```

Comment:
Instead of adding an option, what do you think of merging this with `sequence_style` ?
It is not used in the `before` case and is used only here.

Reply:
Good idea, done.